csv/csv_parse_stream_test.ts

```diff
@@ -57,97 +57,93 @@ Deno.test({
   },
 });
 
-Deno.test({
-  name: "[csv/csv_parse_stream] CsvParseStream with various inputs",
-  permissions: "none",
-  fn: async (t) => {
-    // These test cases were originally ported from Go:
-    // https://github.com/golang/go/blob/go1.12.5/src/encoding/csv/
-    // Copyright 2011 The Go Authors. All rights reserved. BSD license.
-    // https://github.com/golang/go/blob/master/LICENSE
-    const testCases = [
-      {
-        name: "CRLF",
-        input: "a,b\r\nc,d\r\n",
-        output: [["a", "b"], ["c", "d"]],
-      },
-      {
-        name: "BareCR",
-        input: "a,b\rc,d\r\n",
-        output: [["a", "b\rc", "d"]],
-      },
-      {
-        name: "NoEOLTest",
-        input: "a,b,c",
-        output: [["a", "b", "c"]],
-      },
-      {
-        name: "Semicolon",
-        input: "a;b;c\n",
-        output: [["a", "b", "c"]],
-        separator: ";",
-      },
-      {
-        name: "MultiLine",
-        input: `"two
+// These test cases were originally ported from Go:
+// https://github.com/golang/go/blob/go1.12.5/src/encoding/csv/
+// Copyright 2011 The Go Authors. All rights reserved. BSD license.
+// https://github.com/golang/go/blob/master/LICENSE
+const testCases = [
```

Comment:
I don't mind this change but I think it should be in a separate PR. It's best to keep PRs as lean and focused as possible.

Reply:
Ok, I'll revert the change.